dataAccessObjects.py

```diff
@@ -237,6 +237,11 @@ def addLikeCount(self,targetId):
         sql = 'UPDATE image_data SET img_like_count=img_like_count+1 WHERE img_id="{targetId}"'.format(targetId=str(targetId))
         ret = self.db.cmd(sql)
 
+    def getImgCheck(self,imgId):
+        sql = 'SELECT user_id, type_id FROM image_data WHERE img_id="{imgId}"'.format(imgId=imgId)
+        ret = db.query(sql)
```

Comment:
Using the following code is better
```
if len(ret) and len(ret[0]) == 2:
    imgInfo = dict(userId=ret[0][0],typeId=ret[0][1])
    return imgInfo
else:
    return None
```
And change the code in `server_api.py` ( don't forget to check the `imgInfo` is not `None` )